src/modules/geo/GoActive.Modules.Geo.Domain/ValueObjects/Title.cs

```diff
@@ -9,6 +9,15 @@ public record Title
     public static Title FromValue(string value)
     {
         ArgumentException.ThrowIfNullOrWhiteSpace(value);
-        return new(value);
+
+        if(value.StartsWith(" ") || value.EndsWith(" "))
+            throw new ArgumentException("Title value shouldn't contains heading and trailing white spaces.");
```

Comment:
nit: better to use overloaded ArgumentException(string? message, string? paramName)